external/CMakeLists.txt

```diff
@@ -1,4 +1,26 @@
 # The list of files is drived from: breakpad/Makefile.am
+cmake_minimum_required(VERSION 3.12)
+project(sentry_breakpad)
+
+set(CMAKE_CXX_STANDARD 11)
+
+if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
+    set(LINUX TRUE)
+endif()
+
+# set static runtime, if enabled
+if(SENTRY_BUILD_RUNTIMESTATIC AND MSVC)
+    set(CMAKE_MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>")
+endif()
+
+if(LINUX)
+    option(BREAKPAD_PIC "Build breakpad as position independent code" ON)
+else()
+    set(BREAKPAD_PIC ON)
+endif()
+if(BREAKPAD_PIC)
+    set(CMAKE_POSITION_INDEPENDENT_CODE ON)
```

Comment:
```suggestion
    set(CMAKE_POSITION_INDEPENDENT_CODE ON)
else()
    set(CMAKE_POSITION_INDEPENDENT_CODE OFF)
```

Reply:
`set(CMAKE_POSITION_INDEPENDENT_CODE ${BREAKPAD_PIC})`

Reply:
It's a trade off between setting `CMAKE_POSITION_INDEPENDENT_CODE` once,
or setting the `POSITION_INDEPENDENT_CODE` property once for each target.

I thought the first was less error prone.